src/main/java/racing/input/InputView.java

```diff
@@ -1,9 +1,14 @@
 package racing.input;
 
+import racing.exception.InvalidNumberException;
+
+import java.util.InputMismatchException;
 import java.util.Scanner;
 
 public class InputView {
 
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
추출 잘 해주셨습니다 👍 
컨벤션에 따라 대문자로 수정되면 좋을 것 같아요 🙏 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```